src/tree/registries/gitLab/GitLabRepositoryTreeItem.ts

```diff
@@ -17,6 +17,10 @@ export class GitLabRepositoryTreeItem extends RemoteRepositoryTreeItemBase {
     private _nextLink?: string;
 
     public constructor(parent: GitLabProjectTreeItem, id: string, name: string) {
+        if (!name) {
+            name = parent.label;
+        }
+
         super(parent, name);
```

Comment:
An alternative to the `if` block would be this:
`super(parent, name || parent.label)`

This is saying that if `name` is any falsy value (`undefined`, empty string), then use `parent.label`.

Reply:
Oh, it really looks much better. :+1: I'll refactor this.